src/__tests__/core/block.transactionFilter.test.ts

```diff
@@ -355,6 +355,27 @@ describe('transactionFilter on Block->transactions', async () => {
     const result = await execQuery(query);
     expect(result).toEqual(expected);
   });
+
+  test('withLogs: true/false values', async () => {
+    const query = `
+      {
+        block(number: 5000000) {
+          transactionCount
+          noLogs: transactions(filter: { withLogs: false }) {
+            index
+          }
+          withLogs: transactions(filter: { withLogs: true }) {
+            index
+          }
+        }
+      }
+    `;
+
+    const result = await execQuery(query);
+    expect(result.errors).toBeUndefined();
+    const logSum = result.data.block.noLogs.length + result.data.block.withLogs.length;
```

Comment:
So all transactions in this case have logs? And none have "nologs" thus it should just be transaction count + 0?

Reply:
There are some transactions with logs, and some transactions without logs. And the sum of their counts must == the total transaction count.

Reply:
Ah I see. Makes sense :)